### PR TITLE
Update MeRS page for V2 release

### LINKED ISSUE #1098: Remove redundant sentence in MERS guide.
**Describe the error/improvement to an existing document or image**

Remove redundant sentence under Overview.  Under Overview, the first sentence of second para. is a duplicate of first sentence of the page. 

Combine the paragraphs into one after making this change.  

**Screenshots**
NA

### LINKED ISSUE #1142: Minor MeRs doc fix.
In this sentence:

The MeRS provides a pre-built Docker image available on DockerHub, which includes instructions on build the image from source. 

I believe the word "build" should be "building".

**Additional context**
https://docs.01.org/clearlinux/latest/guides/stacks/mers.html

### LINKED ISSUE #1159: Update Media Reference Stack documentation for new release
**Describe the error/improvement to an existing document or image**
This issue is to track updates to the MeRS document for the V2 release. 

**Additional context**
This will also target fixes for:
https://github.com/clearlinux/clear-linux-documentation/issues/1142
https://github.com/clearlinux/clear-linux-documentation/issues/1098

### PR DESCRIPTION
**Pull request by puneetse**
The PR updates the MeRS page with information about the V2 release, examples, information on adding AOM support, and updates to new Intel Stacks locations.

Closes #1159
Closes #1142
Closes #1098

### CODE REVIEW

File: source/guides/stacks/mers.rst [hunk 1]
```diff
@@ -417,4 +510,54 @@ using pre-trained models and sample video files.
...
+#. Use the :command:`docker build` command to build a local copy of the
+   MeRS container image tagged as *aom*.
+
+   .. code-block:: bash
+
+      docker build --no-cache -t sysstacks/mers-clearlinux:aom .
+
+Once the build has completed successfully, the local image can be used
+following the same steps in this tutorial by substituting the image name with
+*sysstacks/mers-clearlinux:aom*.
```
Comment (luisfponce) [inline source/guides/stacks/mers.rst:559]: 👍

File: source/guides/stacks/mers.rst [hunk 2]
```diff
@@ -104,58 +148,58 @@ Components of the |MERS| include:
...
       <https://docs.docker.com/network/proxy/#configure-the-docker-client>`_
       documentation for more details.
       
-#. Once you have downloaded the image, run it with:
+#. Once you have downloaded the image, run it using the following command:
 
    .. code-block:: bash
 
-      docker run -it clearlinux/stacks-mers
+      docker run -it sysstacks/mers-clearlinux
```
Comment (lsandov1) [inline source/guides/stacks/mers.rst:183]: @puneetse please include the parameters ` --device=/dev/dri -e QSV_DEVICE=/dev/dri/renderD128 ` before the image name

File: source/guides/stacks/mers.rst [hunk 3]
```diff
@@ -16,24 +17,42 @@ your own |MERS| container image, and use the reference stack.
...
+<https://github.com/intel/stacks>`_ for information and download links for the
+different versions and offerings of the stack.
+
+* MeRS V0.2.0 release announcement including media processing on GPU and
+  analytics on CPU. 
+
+* MeRS V0.1.0 including media processing and analytics CPU.
+
+* `MeRS Release notes on Github*
+  <https://github.com/intel/stacks/blob/master/mers/releasenotes.md>`_ for the
```
Comment (lsandov1) [inline source/guides/stacks/mers.rst:47]: @puneetse this link is wrong, please confirm @luisfponce

File: source/guides/stacks/mers.rst (hunk 2 above)
Comment (puneetse) [reply source/guides/stacks/mers.rst:183]: The `docker run` with those parameters explained further down in the document in the examples. The idea is to start simple and build on complexity.

File: source/guides/stacks/mers.rst [hunk 4]
```diff
@@ -16,24 +17,42 @@ your own |MERS| container image, and use the reference stack.
...
+<https://github.com/intel/stacks>`_ for information and download links for the
+different versions and offerings of the stack.
+
+* MeRS V0.2.0 release announcement including media processing on GPU and
+  analytics on CPU. 
+
+* MeRS V0.1.0 including media processing and analytics CPU.
+
+* `MeRS Release notes on Github*
+  <https://github.com/intel/stacks/blob/master/mers/NEWS.md>`_ for the
```
Comment (luisfponce) [inline source/guides/stacks/mers.rst:47]: 👍

**Comment by luisfponce** [comment]
LGTM

**Comment by puneetse** [comment]
@mvincerx this is ready to merged ASAP please.

**Comment by mvincerx** [comment]
LGTM.

**Pull request merged**